int64 camera_num
---
int64 count
